README.md

```diff
@@ -1,2 +1,22 @@
 # hw01
-Repository for first CFSS homework assignment
+Repository for first CFSS homework assignment   
+
+## HELLO MY NAME IS...   
+
+Nicholena Honors
+
+*this stuff is tough*  
+
+**but I'm getting by**
+
+### I have a ton more to learn clearly...  
+
+#For breakfast I had...
+
+* [TEA](http://g3yxi3w953w3xjjaj1xcbuay.wpengine.netdna-cdn.com/wp-content/uploads/2015/01/Dollarphotoclub_68652169-e1422257422160.jpg)
+* [EGGS](http://www.incredibleegg.org/wp-content/themes/incredibleegg/assets/images/facts-left-egg.png)
+
+
+I found the rMarkdown help and followed the cues there. I struggled at first with seperating lines but realized that and extra spacing line would work after just messing around for a bit.
+
+I'm still uncertain if there is a way to simply view the output in RStudio...I hit "Knit to HTML", that's the only way I've found to see output.
```

Comment:
Yup, that's how to do it in R Studio. Or just commit and push to Github, then see the rendered version online. But if you want to view it locally "Knitting" is the best way